### ISSUE TITLE
loading particle sets increases rank of  linked attributes 

### ISSUE THREAD

**Issue by ipelupessy** [amusecode/amuse#36]
see the following example:

```
    from amuse.datamodel import Particles
    from amuse.io import write_set_to_file,read_set_from_file
    p=Particles(10)
    q=Particles(3)
    q[0].link=p[1]
    q[1].link=p[3]
    q[2].link=p[4]
    print q.link
    write_set_to_file(q,"test","amuse",version=2)
    r=read_set_from_file("test","amuse")
    print r.link
```

first print gives [..] while second one gives [[...]]

**Comment by rieder** [comment]
Could this be due to the support for multiple timestamps in the savefile?

**Comment by ipelupessy** [comment]
this actually breaks some tests in combination with the new default storage version, e.g. test_smalln.py test16

**Comment by ipelupessy** [comment]
fixed by commit 368b2d48d842635e67da6d2ad3e6c317cdfee493
